tests/README.md

```diff
@@ -0,0 +1,62 @@
+# Splunk OpenTelemetry Collector Integration Tests and Utilities
+
+To assist in vetting and validating the upstream and Splunk Collector distributions of the Collector, this library
+provides a set of integration tests and associated utilities.  The general testing pattern this project is geared toward
+is:
+
+1. Building the Collector (`make otelcol` or `make all`)
+1. Defining your expected [resource metric content](./testutils/README.md#resource-metrics) as a yaml file
+([see example](./testutils/testdata/resourceMetrics.yaml))
+1. Spin up your target resources as docker containers (TODO)
+1. Stand up an in-memory OTLP Receiver and metric sink capable of detecting if/when desired data are received (TODO).
+1. Spin up your Collector as a subprocess configured to report to this OTLP receiver (TODO)
+  
+...but if you are interested in something else enhancements and contributions are a great way to ensure this library
+is more useful overall.
+
+At this time only limited metric content is supported.  If you need additional metric functionality or trace/log
+helpers, please don't hesitate to contribute!
+
+```go
+package example_test
+
+import (
+	"context"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/signalfx/splunk-otel-collector/tests/testutils"
+)
+
+func TestMyExampleComponent(t *testing.T) {
+	expectedResourceMetrics, err := testutils.LoadResourceMetrics(
+		path.Join(".", "testdata", "my_resource_metrics.yaml"),
+	)
+	require.NoError(t, err)
+	require.NotNil(t, expectedResourceMetrics)
+
+	// combination OTLP Receiver consumertests.MetricsSink
+	otlp, err := testutils.NewOTLPReceiverSink().WithEndpoint("localhost:23456").Build()
+	require.NoError(t, err)
+	require.NoError(t, otlp.Start())
+
+	defer func() {
+		require.Nil(t, otlp.Shutdown())
+	}()
+
+	myContainer := testutils.NewContainer().WithImage("someTarget").Build()
```

Comment:
What's "someTarget"? Is it Docker image name?

Reply:
yes, `WithImage()` take an image name